import-export-cli/docs/apictl_get_policies_api.md

```diff
@@ -0,0 +1,44 @@
+## apictl get policies api
+
+Display a list of API Policies in an environment
+
+### Synopsis
+
+Display a list of API Policies in the environment specified by the flag --environment, -e
+
+```
+apictl get policies api [flags]
+```
+
+### Examples
+
+```
+apictl get policies api -e production
+apictl get policies api -e prod --all
+apictl get policies api -e prod -l 10
+apictl get policies api -e prod
```

Comment:
Do we need both lines 16 and 19?

Reply:
Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)